plainly-plugin/src/ui/App.tsx

```diff
@@ -58,7 +101,18 @@ export function App() {
         </p>
       </Banner>
 
-      <div className="flex-1 overflow-y-auto">{route}</div>
+      <div className="flex-1 overflow-y-auto">
```

Comment:
hmm, I don't get this. Why anyone should reload the extension if we can switch the context in the background?